Cargo.toml

```diff
@@ -13,18 +13,18 @@ categories = [
 keywords = ["tracing", "opentelemetry", "jaeger", "zipkin", "async"]
 license = "MIT"
 edition = "2021"
-rust-version = "1.70.0"
+rust-version = "1.75.0"
 
 [features]
 default = ["tracing-log", "metrics"]
 # Enables support for exporting OpenTelemetry metrics
 metrics = ["opentelemetry/metrics","opentelemetry_sdk/metrics", "smallvec"]
 # Enables experimental support for OpenTelemetry gauge metrics
-metrics_gauge_unstable = ["opentelemetry/otel_unstable"]
```

Comment:
Note: This feature was necessary with opentelemetry 0.22, still in use when #129 was merged, but already obsolete with opentelemetry 0.23, so that it was not actually necessary in any tracing-opentelemetry release.